examples/index.html

```diff
@@ -1,29 +1,33 @@
-<!doctype html>
```

Comment:
same — there's no functional change — this is just the result form running prettier on this file.